docs/IE_DG/Extensibility_DG/AddingNGraphOps.md

```diff
@@ -1,61 +1,63 @@
 # Custom nGraph Operation {#openvino_docs_IE_DG_Extensibility_DG_AddingNGraphOps}
 
-Inference Engine Extension API allows to register operation sets (opsets) with custom nGraph operations, it allows to support Networks with unknown operations.
+Inference Engine Extension API enables you to register operation sets (opsets) with custom nGraph operations and to support Networks with unknown operations.
```

Comment:
May be change to something like this:

`Inference Engine Extension API enables you to register operation sets (opsets) with custom nGraph operations to support models with operations which OpenVINO does not support out of the box.`